themes/bootstrap3/templates/RecordDriver/SolrOverdrive/hold.phtml

```diff
@@ -111,149 +129,138 @@
                             '%%numberOfHolds%%' => $result->data->hold->numberOfHolds,
                         ]
                     );
-                endif;?>
+                endif; ?>
             </div>
         </div>
         <?php else: ?>
         <div class="alert alert-danger">
-            <?=$this->icon('overdrive-warning') ?>
-            <div class="moreInfo"><?=$result->msg?></div>
+            <div class="moreInfo"><?=$this->escapeHtml($result->msg)?></div>
         </div>
 
-        <?php endif;?>
-
-    <?php elseif ($action == 'cancelHoldConfirm'): ?>
-            <form class="" method="POST" name="cancelHold" data-lightbox-onclose="VuFind.refreshPage">
+        <?php endif; ?>
 
-                <div class="form-group pull-right">
-                    <input name="doAction" value="cancelHold" type="hidden">
-                    <input class="btn btn-primary" name="cancelHold" value="<?=$this->transEscAttr('od_but_cancel_hold'); ?>" type="submit">
-                </div>
-            </form>
+    <?php elseif ($action == 'editHoldEmailConfirm'): ?>
+        <form class="" method="POST" name="editHoldEmail" data-lightbox-onclose="VuFind.refreshPage">
+            <div class="form-group">
+                <label class="control-label"><?=$this->transEsc('Email Address')?></label>
+                <input id="email" name="email" value="<?=$this->escapeHtmlAttr($result->data->hold->emailAddress)?>" size="" class="form-control" type="text">
 
+            </div>
+            <div class="form-group">
+                <input name="doAction" value="editHoldEmail" type="hidden">
+                <input class="btn btn-primary" name="editHoldEmail" value="<?=$this->transEscAttr('Submit')?>" type="submit">
+            </div>
+        </form>
 
-    <?php elseif ($action == 'returnTitleConfirm'): ?>
-        <form class="" method="POST" name="returnTitle" data-lightbox-onclose="VuFind.refreshPage">
-            <div class="pull-right">
+    <?php elseif ($action == 'editHoldConfirm'): ?>
+        <form class="" method="POST" name="editHold" data-lightbox-onclose="VuFind.refreshPage">
 
-                <input name="doAction" value="returnTitle" type="hidden">
-                <input class="btn btn-primary" name="returnTitle" value="<?=$this->transEscAttr('od_but_return'); ?>" type="submit">
+            <div class="form-group">
+                <label id="select-suspension-label"><?=$this->transEsc('od_susp_hold_confirm')?></label>
+                <select class="form-control" name="suspendValue" id="select-suspension" data-open="false">
+                    <option value="0" selected=""><?=$this->transEsc('od_susp_asap')?></option>
+                    <?php $durations = $this->config()->get('Overdrive')->Overdrive->suspendDurations; ?>
```

Comment:
Is it worth having a default fallback in case the config is unset, e.g.

```
<?php $durations = $this->config()->get('Overdrive')->Overdrive->suspendDurations ?? [7, 14, 21, 28, 60, 90, 180]; ?>
```

That way outdated configurations won't break the functionality. If you do it here, you'll also need to do it in the other place that displays the menu.